src/test/java/org/sagebionetworks/bridge/spring/controllers/ParticipantFileControllerTest.java

```diff
@@ -0,0 +1,146 @@
+    ParticipantFile persisted;
+
+    @BeforeMethod
+    public void beforeMethod() {
+        MockitoAnnotations.initMocks(this);
+
+        session = new UserSession();
+        session.setAppId(TEST_APP_ID);
+        StudyParticipant participant = new StudyParticipant.Builder().withId("test_user").build();
+        session.setParticipant(participant);
+        doReturn(session).when(controller).getAuthenticatedAndConsentedSession();
+        doReturn(mockRequest).when(controller).request();
+        doReturn(mockResponse).when(controller).response();
+
+        persisted = ParticipantFile.create();
+        persisted.setFileId("file_id");
+        persisted.setUserId("test_user");
+        persisted.setCreatedOn(TestConstants.TIMESTAMP);
+        persisted.setAppId("api");
+        persisted.setMimeType("dummy-type");
+    }
+
+    @Test
+    public void verifyAnnotations() throws Exception {
+        assertCrossOrigin(ParticipantFileController.class);
+        assertGet(ParticipantFileController.class, "getParticipantFile");
+        assertGet(ParticipantFileController.class, "getParticipantFiles");
+        assertPost(ParticipantFileController.class, "createParticipantFile");
+        assertDelete(ParticipantFileController.class, "deleteParticipantFile");
+    }
+
+    @Test
+    public void getParticipantFiles() {
+        ParticipantFile file = ParticipantFile.create();
+        file.setUserId("test_user");
+        file.setFileId("file_id");
+        ForwardCursorPagedResourceList<ParticipantFile> page = new ForwardCursorPagedResourceList<>(
+                ImmutableList.of(file), null);
+        when(mockFileService.getParticipantFiles(eq("test_user"), isNull(), anyInt())).thenReturn(page);
+
+        ForwardCursorPagedResourceList<ParticipantFile> result = controller.getParticipantFiles(null, "5");
+        assertSame(page, result);
+
+        verify(mockFileService).getParticipantFiles(eq("test_user"), isNull(), eq(5));
+    }
+
+    @Test
+    public void getParticipantFile() {
+        when(mockFileService.getParticipantFile(eq("test_user"), eq("file_id"))).thenReturn(persisted);
+        ParticipantFile result = controller.getParticipantFile("file_id");
+        assertSame(result, persisted);
+
+        verify(mockFileService).getParticipantFile("test_user", "file_id");
+        verify(mockResponse).setHeader(eq("Location"), eq(result.getDownloadUrl()));
+    }
+
+    @Test
+    public void createParticipantFile() throws Exception {
+        ParticipantFile newFile = ParticipantFile.create();
+        newFile.setUserId("test_user");
+        newFile.setFileId("file_id");
+        newFile.setMimeType("dummy-type");
+        mockRequestBody(mockRequest, newFile);
+        when(mockFileService.createParticipantFile(any(), any(), any())).thenAnswer(i ->
+        {
+            newFile.setAppId(i.getArgument(0));
+            newFile.setCreatedOn(TestConstants.TIMESTAMP);
+            return newFile;
+        });
```

Comment:
It's not necessary to mock these behaviors. We want to test the behavior of the Controller, not the Service.

Also, you need to verify the inputs to the service.createParticipantFile() call.